tedana/decomposition/ica.py

```diff
@@ -87,7 +88,7 @@ def tedica(
     else:
         raise ValueError("The selected ICA method is invalid!")
 
-    return mixing, fixed_seed
+    return mixing, fixed_seed, c_labels, similarity_t_sne
```

Comment:
I think I see why a bunch of the integration tests are failing. You added code to add the robustica QC figures and measures to the report & that code is running even when the robustica option is not selected. Here's one example where you are returning `c_labels` and `similarity_t_sne` but those won't exist if `ica_method == "fastica"`

Reply:
Yes, I know. I was only waiting on feedback on the 5 echo before I made these changes. I thought the plot itself was the main contribution here and so I wanted to make sure folks were happy with that.